app/helpers/elmo_form_helper.rb

```diff
@@ -4,8 +4,21 @@ module ElmoFormHelper
   # renders a form using the ElmoFormBuilder
   def elmo_form_for(obj, *args, &block)
     options = args.extract_options!
-    form_for(obj, *(args << options.merge(:builder => ElmoFormBuilder,
-      :html => {:class => "#{obj.class.model_name.singular}_form elmo_form #{options[:style]}"})), &block)
+
+    args << options.deep_merge(
```

Comment:
This is a bit cryptic. Perhaps a comment here. Also perhaps construct the param to deep_merge on the line before so that it is not so heinous. :)